tests/system/apmserver.py

```diff
@@ -14,17 +18,99 @@ def setUpClass(cls):
         cls.beat_path = "../../"
         super(BaseTest, cls).setUpClass()
 
+    def get_payload(self):
```

Comment:
this is the `transaction` payload, I suggest to either rename the method to `get_transaction_payload` or pass in a string indicating which payload to return.